.github/workflows/deploy-github-page.yml

```diff
@@ -71,12 +71,14 @@ jobs:
       - name: Jupyter Lite integration 
         shell: bash -l {0}
         run: |
-          micromamba create -n xeus-lite-host jupyterlite-core
+          micromamba create -n xeus-lite-host jupyterlite-core jupyter_server notebook
```

Comment:
Is notebook really required?

Reply:
Not really but I think was added earlier by a contributor as an alternative to lab, hence the change is present there. Keeping it for now and can be removed subsequently if not adding value.

Reply:
> as an alternative to lab

Installing notebook will have no impact on this concerning JupyterLite.

It's the `jupyter lite build --app notebook --app lab` which impacts which kinds of UIs are installed.

JupyterLite already defaults to providing the notebook UI actually, you'll only need to point to a different URL. e.g. for xeus-r you can go on https://jupyter-xeus.github.io/xeus-r/tree instead of https://jupyter-xeus.github.io/xeus-r/lab

Reply:
Ohh okay wow, I didn't know we could frame the url like this. Getting rid of it !